slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -332,11 +333,13 @@ bool SLAMServiceImpl::ExtractPointCloudToBuffer(std::stringbuf &buffer) {
     return has_points;
 }
 
-void SLAMServiceImpl::ProcessData() {
+void SLAMServiceImpl::RunSLAM() {
```

Comment:
Swapped function names `RunSLAM` with `ProcessData` (now `ProcessDataAndStartSavingMaps`) cause they were more descriptive that way.